lib/perl/Genome/InstrumentData/AlignmentResult.pm

```diff
@@ -1661,8 +1660,14 @@ sub revivified_alignment_bam_file_paths {
         return @bams;
     }
 
-    my $revivified_bam = File::Spec->join($p{disk_allocation}->absolute_path, 'all_sequences.bam');
-    my $merged_bam    = $self->get_merged_bam_to_revivify_per_lane_bam;
+    my $temp_allocation = $self->_get_temp_allocation($self->output_dir);
+    UR::Context->process->add_observer(
```

Comment:
I believe that it is worth investigating using Scope::Guard with a future PR, but I went ahead and merged this PR in the interests of getting something functional in place since this was blocking Allison and Susi's work with QC.